modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/K6ClientCodegen.java

```diff
@@ -506,33 +634,47 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
                                 break;
                         }
                     }
-                } catch (NullPointerException e) {
+                } catch (NullPointerException ignored) {
```

Comment:
let's log this exception and know what's broken

Reply:
Addressed.